pcdsdevices/mirror.py

```diff
@@ -293,8 +293,42 @@ class HardOffsetMirror(Device, BaseInterface):
     name : ``str``
         Alias for the device
     """
-    not_implemented = Cpt(SignalRO, name="Not Implemented",
-                          value="Not Implemented", kind='normal')
+    # Motor components: can read/write positions
+    y_up = Cpt(EpicsSignal, ':MMS:YUP', name='y_up', kind='normal')
+    y_dwn = Cpt(EpicsSignal, ':MMS:YDWN', name='y_dwn', kind='config')
+    x_up = Cpt(EpicsSignal, ':MMS:XUP', name='x_up', kind='normal')
+    x_dwn = Cpt(EpicsSignal, ':MMS:XDWN', name='x_dwn', kind='config')
+    pitch = Cpt(EpicsSignal, ':MMS:PITCH', name='pitch', kind='normal')
+    bender = Cpt(EpicsSignal, ':MMS:BENDER', name='bender', kind='normal')
+
+    # Gantry components
+    gantry_x = Cpt(EpicsSignalRO, ':GANTRY_X_RBV', name='gantry_x',
```

Comment:
Do not supply names here, `ophyd` will automatically fill these in from the attribute name `gantry_x` on the left, prepending with the full device name e.g. `mr1l0_gantry_x`

Reply:
Resolved in commit bb4a164